marklogic-data-hub/src/main/java/com/marklogic/hub/step/StepDefinition.java

```diff
@@ -17,13 +17,13 @@
 package com.marklogic.hub.step;
 
 import com.fasterxml.jackson.databind.JsonNode;
-import com.marklogic.hub.step.impl.StepImpl;
+import com.marklogic.hub.step.impl.*;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Map;
 
-public interface Step {
+public interface StepDefinition {
 
     enum StepType {
```

Comment:
Should the enum be called StepDefinitionType ?

Reply:
Yep missed that.
Done https://github.com/marklogic/marklogic-data-hub/pull/2207/commits/168804fa9b372d529eb83717f74637bdc401254e